doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -0,0 +1,1209 @@
+local html = import 'html.libsonnet';
```

Comment:
How do you feel about this file being pure data and then having it exported from comments in the code?

Reply:
Yeah, I think this is the right way. 

In a sense this is pure data already – there are no functions or hidden fields in the output object. Any function calls here are just for readability convenience. In other words you can evaluate it and serialize it to JSON and use that instead of using `stdlib-content.jsonnet` directly.